src/audio/crossover/crossover.c

```diff
@@ -297,7 +297,7 @@ static int crossover_init(struct processing_module *mod)
 	struct comp_dev *dev = mod->dev;
```

Comment:
Fix patch title, not just IIR.

Reply:
@singalsu instead of modifying all the components, would it make sense to change the cfg->size to subtract the sizeof(struct ipc4_base_module_cfg) here?
https://github.com/thesofproject/sof/blob/42c03acbacc9c04a41862e26e31f1b9e09eff736/src/audio/module_adapter/module_adapter_ipc4.c#L51

```
dst->size = cfgsz - sizeof(cfg->base_cfg);
```

Reply:
This would be simplest, good idea! But I'm not able to test all the components, there's several without testbench or cmocka. I recall some components use the extended base module config, for pins. Need to check those can work correctly with this.

Reply:
I checked init() of every component. It looks like selector and src components need for their init() function compensation of sizeof(cfg->base_cfg) subtract from size but it's smaller than changing every component so I'll do it that way.